src/parameterizations/vertical/MOM_diabatic_driver.F90

```diff
@@ -145,8 +146,8 @@ module MOM_diabatic_driver
                                      !! diffusivity of Kd_min_tr (see below) were operating.
   real    :: Kd_BBL_tr               !< A bottom boundary layer tracer diffusivity that
                                      !! will allow for explicitly specified bottom fluxes
-                                     !! [H Z T-1 ~> m2 s-1 or kg m-1 s-1].  The entrainment at the bottom is at
-                                     !! least sqrt(Kd_BBL_tr*dt) over the same distance.
+                                     !! [H2 T-1 ~> m2 s-1 or kg2 m-4 s-2].  The entrainment at the
```

Comment:
NB units should be kg m-1 s-1 ?

Reply:
In this case, Kd_BBL_tr has weird units because it is being used (by multiplying by the timestep and taking a the square root) to set a minimal smoothing thickness for tracers at the bottom.  Kd_BBL_tr is effectively a diffusivity times a density in non-Boussinesq mode.  In this case we are effectively using a prescribed density - the same one (RHO_KV_CONVERT) as is being used to convert diffusivities from [m2 s-1] to [kg m-1 s-1] in non-Boussinesq mode.

Were we being more precise, this density that is being multiplied by a near-bottom diffusivity would probably be a density averaged over a near-bottom thickness, and perhaps that averaging thickness itself would be determined by `sqrt(Kd_BBL * rho_avg)`, but this then becomes an implicit, iteratively solved equation that hardly seems like it would worth the extra complexity and cost.